src/blog/lagom-1.4.3.md

```diff
@@ -0,0 +1,60 @@
+---
+title: Lagom 1.4.3 released!
+date: 2018-03-27
+author_github: renatocaval
+tags: news
+summary: >
+    The Lagom team has released Lagom 1.4.3
+---
+
+The Lagom team has released Lagom 1.4.3. 
+
+This release provides a fix for [#1250](https://github.com/lagom/lagom/issues/1250). 
+
+[#1250](https://github.com/lagom/lagom/issues/1250) only affects Lagom projects making usage of JDBC persistence layer, being it for the journal and/or for read-side tables. 
+
+This release upgrades the versions of the underlying Slick library to `3.2.3` and the `akka-persistence-jdbc` plugin to version `3.3.0`.
+
+## Slick 3.2.3 special note
+
+Since Slick 3.2.2, the `maxConnections` must equal `numThreads`. This is to prevent some situations where a deadlock could occur. A warning will be emitted in case `maxConnections > numThreads` and a runtime exception will be thrown if `maxConnections < numThreads`. Therefore the best possible configuration is `maxConnections == numThreads`. See https://github.com/slick/slick/pull/1855 for detailed information.
```

Comment:
is `https://github.com/slick/slick/pull/1855` converted to a link automagically? If not, can we make it a link please?

Reply:
Done.